docs/content/direct/release.md

```diff
@@ -29,6 +38,8 @@ Making a new kubestellar release requires a contributor to do the following thin
 
 - Edit [the examples document](examples.md) to update the self-references for the coming release.
 
+- Edit [the k3d deploy instructions](deploy-on-k3d.md) to update the self-reference.
+
```

Comment:
k3d is just an example and not part of every release update

Reply:
We need to maintain this document or mark it as a historical point-in-time.

My understanding is that we are working on supporting users who use k3d. @eaepstein ?

Reply:
We haven't decided that. For now we support it through point in time example. We can't support continuously all our examples, we would need to add them into the release test, and validat8each release. This is time consuming

Reply:
If and when the e2e test using k3d is integrated it can be part of the automated test and the write-up significantly simplified. Until then we can add something to the document like "tested on release x.y" and validating with the latest release can be optional.

Reply:
@eaepstein Exactly.  So it should not be part of our release documents (i.e., the need to update the links in the example for every release).  As all the other non-e2e examples we need to include a specific release version on which it was tested.

Reply:
As far as I can tell, currently `deploy-on-k3d.md` does not work as written and produces a state from which the user can not proceed to `examples.md`. This is mainly because of anticipation of laudible changes in naming conventions throughout this project, to remove its overwhelming specialization to infrastructure that comes from `kind`.

I have extended this PR with commits that retreat from than ambition, modifying `deploy-on-k3d.md` to something that _does_ work and produces a state from which a user _can_ proceed to `examples.md`. Yes, with the unsightly fact that the kubeflex hosting context is named "kind-kubeflex" --- because that is baked into `examples.md`. I think that changing the rest of the project contents to remove the dedication to `kind` is a good thing to do but belongs in a separate PR.